doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -115,6 +115,9 @@ Additional scenario XML elements:
 ``guest_flags``:
   Select all applicable flags for the VM.
 
+``severity``:
+  Severity of the guest VM.
+
 ``vcpu_affinity``:
   vCPU affinity map. Each vCPU will pin to the selected pCPU ID. A different vCPU cannot pin to the same pCPU.
```

Comment:
Is this still true with `cpu_sharing`?

Reply:
@deb-intel wouldn't be better to say "will be mapped to" instead of "will pin to"?

Reply:
@gvancuts Yes, I agree.